---
title: LOCALDB_ERROR_UNKNOWN_INSTANCE |Microsoft Docs
ms.custom: ''
ms.date: 03/03/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 67bca4f1-915d-46d4-9293-3742c16930a1
author: stevestein
ms.author: sstein
ms.openlocfilehash: fbc2107811c84b36822a1c8b6fb4bdd5fabbdb41
ms.sourcegitcommit: 216f377451e53874718ae1645a2611cdb198808a
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 07/28/2020
ms.locfileid: "87245971"
---
# <a name="localdb_error_unknown_instance"></a>LOCALDB_ERROR_UNKNOWN_INSTANCE
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>詳細  
  
| 属性 | 値 |
| --------- | ----- |
|製品名|SQL Server|  
|イベント ID|263|  
|イベント ソース|SQL Server Local Database Runtime 12.0|  
|コンポーネント|Local Database Runtime API|  
|メッセージ テキスト|指定したローカル データベース インスタンスは存在しません。|  
  
## <a name="explanation"></a>説明  
 指定したインスタンスは存在しません。  
  
## <a name="user-action"></a>ユーザーの操作  
 指定したインスタンス名が正しいかどうかを確認してください。  
  
  
